### ISSUE TITLE
Discriminator Forward Pass

### ISSUE THREAD

**Issue by nour-alkhatib** [Sxjdwang/TalkLip#10]
I'd like to thank the authors for their incredible effort.

I have a question regarding the discriminator forward pass, specifically the [get_lower_half()](https://github.com/Sxjdwang/TalkLip/blob/7a3ba88185ab221a0b01dbce8beaee70b8e2f076/models/talklip.py#LL178C16-L178C16) function. Shouldn't this function return the lower half along both width and height dimensions? It seems to be doing it along one axis only, so the returned tensor would be of shape: [N, 3, 48, 96]. I would appreciate your clarification on this!

**Comment by Sxjdwang** [comment]
Yes, we only get the lower half along the height dimension. The upper half is provided in inputs and the lower half is synthesized, so that the lower half  [N, 3, 48, 96] is fed to the discriminator.
